packages/core/package.json

```diff
@@ -4218,23 +4218,31 @@
         "clean": "ts-node ../../scripts/clean.ts dist/",
         "copyFiles": "ts-node ./scripts/build/copyFiles.ts",
         "buildScripts": "npm run generateClients && npm run generatePackage && npm run copyFiles",
-        "compile": "npm run clean && npm run buildScripts && tsc -p ./ && webpack --mode development",
-        "compileLite": "tsc -p ./",
-        "watch": "npm run clean && npm run buildScripts && tsc -watch -p ./",
-        "testCompile": "npm run clean && npm run buildScripts && tsc -p ./",
+        "compile": "npm run testCompile && webpack",
```

Comment:
Not a blocker for this PR,  but it's weird that `compile` depends on `testCompile`. And previously `compile` was doing a development build. All of this is confusing. But it's tech debt we've had for a long time.

Reply:
My rationale is that a `testCompile` is enough to get testing working, but a full `compile` is `testCompile` plus anything extra needed for a production compile. I still feel we could simplify things for sure so open to any suggestions.